types/core/rolls/dicePool.d.ts

```diff
@@ -16,14 +16,57 @@
  *   modifiers: ["kh"]
  * });
  * pool.evaluate();
+ *
  * ```
  * @example
- * ```javascript
+ * ```typescript
  * // Construct a DicePool from a string formula
  * let pool = DicePool.fromExpression("{4d6,3d8,2d10}kh");
  * ```
  */
 declare class DicePool {
+  /**
+   * @param modifiers - (default: `[]`)
+   * @param options   - (default: `{}`)
+   * @param rolls     - (default: `[]`)
+   */
+  constructor({ rolls, modifiers, options }?: { rolls?: Array<Roll | number>; modifiers?: string[]; options?: object });
+
+  /**
+   * The elements of a Dice Pool must be Roll objects or numbers
+   */
+  rolls: Array<Roll | number>;
+
+  /**
+   * The string modifiers applied to resolve the pool
+   */
+  modifiers: string[];
```

Comment:
I think this might be possible to type? It's something like keyof DicePool['MODIFIERS']